templates/base.html

```diff
@@ -2,20 +2,76 @@
 
 <!DOCTYPE html>
 <html lang="en-gb">
-<meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```

Comment:
I believe we included that  essential meta tag with previous PR

Reply:
The line with minus sign in the front means the line was removed.

The green line with plus sign in front means the line was added back again.


So this line was removed and then added again. Why? I reformatted it to be more readable.

See my comment below about Prettier.

Reply:
I see there is a backslash at the end of the line (meta tag) as a difference between the two -- :)